commitlint.config.js

```diff
@@ -1,3 +1,8 @@
 module.exports = {
     extends: ['@commitlint/config-conventional'],
+    rules: {
+            'body-max-line-length': [2, 'always', 88],
+            'footer-max-line-length': [2, 'always', 88],
+            'header-max-length': [2, 'always', 72]
+    }
```

Comment:
```suggestion
    rules: {
        'header-max-length': [2, 'always', 72],
    },
```

While I like the idea of setting values for `body-max-line-length` and `footer-max-line-length`, I'm not sure there are ways to set exceptions for these rules, to allow for content such as long URL references, inline code/error examples or other such ["non-prose" text](https://stackoverflow.com/a/17792445).

Otherwise, the suggestion above also formats according to a 4-space indent style (which reminds me that I've got an `eslint` configuration available that could be useful for the `js` files in our formulas).